styles/choose-plan.css

```diff
@@ -1,9 +1,79 @@
 .choose-plan {
- margin-top: 126px;
+  margin-top: 126px;
 }
-.choose-plan__title{
-    text-align: center;
+.choose-plan__title {
+  text-align: center;
 }
-.choose-plan__description{
-    text-align: center;
+.choose-plan__description {
+  text-align: center;
+  margin: 30px 0;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 30px;
+}
+.choose-plan__wrap {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1090px;
+  margin: 0 auto;
+}
+.choose-plan__item {
+  background: #ffffff;
+  border: 2px solid #dddddd;
+  box-sizing: border-box;
+  border-radius: 10px;
+  max-width: 330px;
+  width: 100%;
+  padding: 80px 0 50px 0;
+}
+.choose-plan__item:hover {
+  border: 2px solid #f53838;
+  cursor: pointer;
+}
+.choose-plan__ill-free {
+  display: block;
+  margin: 0 auto;
+}
+.choose-plan__list--item {
+  margin-left: 70px;
+}
+.choose-plan__list {
+  display: flex;
+  margin-bottom: 10px;
+}
+.choose-plan__text {
+  margin-left: 20px;
+  font-size: 14px;
+  line-height: 30px;
+  color: #4f5665;
+}
+.choose-plan__subtitle {
+  text-align: center;
+  font-size: 25px;
+  line-height: 30px;
+  margin: 50px 0 20px 0;
```

Comment:
The block with the price and the button should be stuck to the bottom of the box.

Reply:
++